test/Microsoft.AspNet.Mvc.IntegrationTests/TryValidateModelIntegrationTest.cs

```diff
@@ -140,7 +140,7 @@ public void TryValidateModel_CollectionsModel_ReturnsErrorsForInvalidProperties(
         {
             var controller = new TestController();
             controller.ControllerContext = new ControllerContext(operationContext.ActionContext);
-            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator();
+            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator(operationContext.MetadataProvider);
```

Comment:
Does this indicate you need to rebase or an unrelated change mixed into this PR?